content/cucumber/configuration.md

```diff
@@ -1,13 +1,75 @@
 ---
 title: Cucumber Configuration
-subtitle: cucumber.yml, environment variables
+subtitle: cucumber.yml, type registry
 polyglot: true
 ---
 
+
+# Type Registry Configuration
+
+{{% block "java" %}}
+The type registry is used to configure parameter and data table types. It can be configured by placing an implementation 
+of `cucumber.api.TypeRegistryConfigurer` on the glue path.
+
+
+```java
+public class TypeRegistryConfiguration implements TypeRegistryConfigurer {
+
+    @Override
+    public Locale locale() {
+        return ENGLISH;
+    }
+
+    @Override
+    public void configureTypeRegistry(TypeRegistry typeRegistry) {
+        typeRegistry.defineParameterType(new ParameterType<Integer>(
+            "digit",
+            "[0-9]",
+            Integer.class,
+            new Transformer<Integer>() {
+                @Override
+                public Integer transform(String s) throws Throwable {
+                    return Integer.parseInt(s);
+                }
+            })
+        );
+
+        typeRegistry.defineDataTableType(new DataTableType(
+            ItemQuantity.class,
+            new TableCellTransformer<ItemQuantity>() {
+                @Override
+                public ItemQuantity transform(String s) {
+                    return new ItemQuantity(s);
+                }
+            })
+        );
+    }
+}
+```
+{{% /block %}}
+
+{{% block "ruby" %}}
+ TODO:
```

Comment:
Personally, I don't think we should have "TODO" anywhere in the documentation. If you cannot add this information yourself, please try to find someone with Ruby knowledge who can do it (or help you do it), or (if that fails) create an issue marked "Ruby"/"Help wanted"